### ISSUE TITLE
Can't find data in ExtractIlluminaBarcodes metrics log

### ISSUE THREAD

**Issue by lizcodd** [ewels/MultiQC#1611]
### Description of bug

MultiQC finds the correct log file but can't find the data due to potential input file format change:

MultiQC looks for both `## METRICS CLASS` and `ExtractIlluminaBarcodes` in the same line but this was not output by Picard ExtractIlluminaBarcodes Version:2.26.2

### File that triggers the error

[1_barcode_metrics.log](https://github.com/ewels/MultiQC/files/7832462/1_barcode_metrics.log)

### MultiQC Error log

```console
multiqc .

 /// MultiQC 🔍 | v1.11

|           multiqc | Search path : /scratch/lab/cfDNA/run_results/logs
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 73/73  
|          bargraph | Tried to make bar plot, but had no data
|          bargraph | Tried to make bar plot, but had no data
|            picard | Found 4 ExtractIlluminaBarcodes reports
|           multiqc | Compressing plot data
|           multiqc | Report      : multiqc_report.html
|           multiqc | Data        : multiqc_data
|           multiqc | MultiQC complete
```

**Comment by ewels** [comment]
Thanks for letting me know! The Picard outputs can be pretty variable, it's always a bit of a moving target!

**Comment by ewels** [comment]
Should be fixed in 7a0229c632ffda4b7b6456c460511f54a3062d57 - thanks for reporting!
